dev.Dockerfile

```
@@ -0,0 +1,17 @@
+FROM rustlang/rust:nightly
+
+RUN apt update && \
```

Hadolint (2.14.0)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)